src/components/ShowList.css

```diff
@@ -0,0 +1,15 @@
+.list-view {
```

Comment:
Good job. There are multiple ways to style it and [this](https://thecssworkshop.com/lessons/combinators) can be a good article about combinators.